### ISSUE TITLE
Failed to run /bin/bash: ./scripts/startup.sh: No such file or directory

### ISSUE THREAD

**Issue by ephrin** [ypereirareis/docker-satis#7]
Does it needs to use `-v $(pwd):/app` as a workdir with this repository sources cloned?

**Comment by ypereirareis** [comment]
Hi !

Yes you need as it's said in the doc :smiley: 

```
docker run -it -p 3033:3000 -p 80:80 \
  -v $(pwd):/app \
  -v "${HOME}/.ssh/id_rsa":/var/tmp/id \
  -v /var/tmp/composer:/root/.composer \
  -e PRIVATE_REPO_DOMAIN=toto.tata.tutu.com \
  -e CRONTAB_FREQUENCY="*/10 * * * *" \
  -d ypereirareis/docker-satis
```

**Comment by ephrin** [comment]
Thanks.
$(pwd) for me after reading docs (README.md) is just cwd. And only after reading this line https://github.com/ypereirareis/docker-satis/blob/master/Dockerfile#L74 all becomes clear ;)

**Comment by ephrin** [comment]
Maybe there should be only one outer volume dependency (config.json) and all other files builded in image?

**Comment by ypereirareis** [comment]
Maybe..... feel free to send a PR :smile_cat: No time to do this...

**Comment by ypereirareis** [comment]
Hello !

This problem is now fixed if you use tag 1.0.0 of this image.

**Comment by ephrin** [comment]
Great! 👍
04 сент. 2015 г. 14:53 пользователь "Yannick Pereira-Reis" <
notifications@github.com> написал:
